pkg/sqlite/query.go

```diff
@@ -970,6 +970,16 @@ tip (depth) AS (
       INNER JOIN channel_entry AS skipped_entry
         ON skips_entry.skips = skipped_entry.entry_id
     GROUP BY all_entry.operatorbundle_name, all_entry.package_name, all_entry.channel_name
+),
+merged_properties (bundle_name, merged) AS (
+  SELECT operatorbundle_name, json_group_array(json_object('type', properties.type, 'value', properties.value))
```

Comment:
I'n the [existing code](https://github.com/operator-framework/operator-registry/pull/720/files#diff-5d7535c9d61e38ec8f7cd26e7746b174a76b808fb64e098679f584c2a0986f34L1035), properties / dependencies are only merged by cases of matching name + version + path + channel name, it seems as though this change only considers bundle name. Is there a reason to be concerned that this may change existing user behavior?

Reply:
`name` is the full primary key of `operatorbundle`, so there can't be two rows in that table that have the same name but different path/version. Channel is already a nonissue because each result row is pretty much hung off of a single row in `channel_entry`.